addons/portal/static/src/xml/portal_chatter.xml

```diff
@@ -62,14 +62,16 @@
 
     <t t-name="portal.Chatter.Attachments">
         <div t-if="attachments.length" class="d-flex flex-grow-1 flex-wrap gap-1">
-            <div t-foreach="attachments" t-as="attachment" t-key="attachment_index" class="bg-light p-2 rounded">
-                <div class="o_portal_chatter_attachment position-relative text-center" t-att-data-id="attachment.id">
+            <div t-foreach="attachments" t-as="attachment" t-key="attachment_index" class="bg-light p-2 rounded position-relative">
+                <div class="o_portal_chatter_attachment text-center" t-att-data-id="attachment.id">
```

Comment:
Why did you move the position-relative class here?

When testing it does not seem to make any difference.

Reply:
Never mind, it's the cross alignment, had to re-read the task pad.